geoips/geoips_utils.py

```diff
@@ -445,3 +449,129 @@ def merge_nested_dicts(dest, src, in_place=True):
         raise
     if not in_place:
         return final_dest
+
+
+def expose_geoips_commands(pkg_name=None, test_log=None):
+    """Expose a list of commands that operate in the GeoIPS environment.
+
+    Where, these commands are defined under 'pyproject.toml:[tool.poetry.scripts]',
+    or 'pyproject.toml:[project.entry-points.console_scripts]'
+
+    Parameters
+    ----------
+    pkg_name: str (default = None)
+        - The name of the GeoIPS Plugin Package whose command's will be exposed.
+        - If None, assume this was called via the commandline and retrieve package_name
+          via that manner. Otherwise use the supplied package_name.
+    test_log: logging.Logger (default = None)
+        - If provided, use this log so we can test the output of this function for
+          testing purposes
+    """
+    pkg_name, log = _get_pkg_name_and_logger(pkg_name, test_log)
+    try:
+        toml_path = resources.files(pkg_name) / "../pyproject.toml"
+
+        with open(toml_path, "r") as toml_file:
+            pyproj = toml.load(toml_file)
+            scripts = _get_pyproj_scripts(pyproj)
+            # If scripts were found in this package, expose them
+            if scripts:
+                log.interactive("-" * len(f"Available {pkg_name.title()} Commands"))
```

Comment:
I tend to think this is a clean way of doing it. I've looked through ``tabulate``'s documentation and they don't support added a title to the table unfortunately. I did this for all of the ``geoips list commands`` and just stuck to the formatting I used for those PRs. Happy to hear everyone's thoughts on this though.

Reply:
Since this is a small part of this PR + standard in other PRs I'll go ahead and give my blessing. In general, I like to abstract functionality that outputs to the terminal to a separate function that handle formatting. I think it makes future refactoring easier, and segments functionality